modules/user/src/main/TotpSecret.scala

```diff
@@ -23,27 +25,29 @@ case class TotpSecret(secret: Array[Byte]) extends AnyVal {
     hmac.init(new SecretKeySpec(secret, "RAW"))
     val hash = hmac.doFinal(msg)
 
-    val offset = hash(hash.length - 1) & 0xf
+    val offset = hash(19) & 0xf
```

Comment:
i think the previous version or maybe `hash.last` better shows what's happening here

Reply:
Yeah that works too.  We're already using the hash size when we index into the array at `offset` so I thought I'd just hard code this, but .last works too